Automattic/newspack-theme
# feat: add Bluesky to the profile fields

### All Submissions:






### Changes proposed in this Pull Request:



Adds the Bluesky profile field.

Note. There's also another filter in Yoast that I was trying, that looks like it has to do with it, but I don't see it having any effect, so I removed it. It's this:

```
add_filter(
'wpseo_person_social_profile_fields',
function( $profile_fields ) {
	if ( ! array_key_exists( 'bluesky', $profile_fields ) ) {
		$profile_fields['bluesky'] = 'get_non_valid_url';
	}
	return $profile_fields;
}
);
```

### How to test the changes in this Pull Request:

1. Edit an author profie
2. Confirm you see a new field for Bluesky profile
3. Add a url starting with https://bsky.app/profile/
4. Visit the author archive page
5. Confirm you see the Bluesky link with the correct icon


### Other information:
